### PR TITLE
Feat/update upload status popups

### PR DESCRIPTION
**Pull request by alex-silverman**

### CODE REVIEW

File: tests/testthat/_snaps/linux-4.4/fsa/test-fsa-001.json [hunk 1]
```diff
@@ -296,7 +296,7 @@
         "Your HMIS CSV Export should contain - with identical, case-sensitive spelling - only the columns specified in the columns.csv file. Please remove any extra columns and make sure all columns represented in the HMIS CSV Format Specifications are present.",
         "Your HMIS CSV Export should contain - with identical, case-sensitive spelling - only the columns specified in the columns.csv file. Please remove any extra columns and make sure all columns represented in the HMIS CSV Format Specifications are present.",
         "Per the HMIS CSV Format Specifications, all PersonalIDs in the Enrollment file should have a record in the Client file.",
-        "Per the HMIS CSV Format Specifications, all ProjectIDs in the Enrollment file should have a record in the Project file.",
+        "Per the HMIS CSV Format Specifications, all ProjectIDs in the Enrollment file should have a record in the Proect file.",
```
Comment (kiadso) [inline tests/testthat/_snaps/linux-4.4/fsa/test-fsa-001.json:299]: (minor typo here) ^

File: tests/testthat/_snaps/linux-4.4/fsa/test-fsa-002.json [hunk 2]
```diff
@@ -320,7 +320,7 @@
         "Your HMIS CSV Export should contain - with identical, case-sensitive spelling - only the columns specified in the columns.csv file. Please remove any extra columns and make sure all columns represented in the HMIS CSV Format Specifications are present.",
         "Your HMIS CSV Export should contain - with identical, case-sensitive spelling - only the columns specified in the columns.csv file. Please remove any extra columns and make sure all columns represented in the HMIS CSV Format Specifications are present.",
         "Per the HMIS CSV Format Specifications, all PersonalIDs in the Enrollment file should have a record in the Client file.",
-        "Per the HMIS CSV Format Specifications, all ProjectIDs in the Enrollment file should have a record in the Project file.",
+        "Per the HMIS CSV Format Specifications, all ProjectIDs in the Enrollment file should have a record in the Proect file.",
```
Comment (kiadso) [inline tests/testthat/_snaps/linux-4.4/fsa/test-fsa-002.json:323]: (again, and I'm seeing in a few other places)

**Review by kiadso — APPROVED** [review]
I ran this on a few different "bad" files on ACE and found that everything worked nicely and the popups make sense to me. I didn't see anything unexpected. Also checked demo mode, also looks good. Thank you @alex-silverman and @alyssandrichik!

File: tests/testthat/_snaps/linux-4.4/fsa/test-fsa-002.json (hunk 2 above)
Comment (alex-silverman) [reply tests/testthat/_snaps/linux-4.4/fsa/test-fsa-002.json:323]: fixed in EvaChecks.csv

File: tests/testthat/_snaps/linux-4.4/fsa/test-fsa-001.json (hunk 1 above)
Comment (alex-silverman) [reply tests/testthat/_snaps/linux-4.4/fsa/test-fsa-001.json:299]: fixed in EvaChecks.csv

**Pull request merged**